### PR TITLE
setup project configuration

### PR DESCRIPTION
**Pull request by JuanManu59**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+js* @holgiosalos @danielgalvis98 @kliver98 @AlejaGonzalezV @NicolasB2 @manuelq12 @Valeryibarra @veronicatofino
```
Comment (Valeryibarra) [inline .github/CODEOWNERS]: js at begin, what that means?

Comment (JuanManu59) [reply .github/CODEOWNERS]: my mistake, I copied and pasted

Comment (Valeryibarra) [reply .github/CODEOWNERS]: no worries :3

Comment (JuanManu59) [reply .github/CODEOWNERS]: resolved

**Review by Valeryibarra — APPROVED** [review]

**Pull request merged**